app/services/onlinetesting/Phase2TestService.scala

```diff
@@ -95,22 +96,12 @@ trait Phase2TestService extends OnlineTestService with Phase2TestConcern with Sc
     }
   }
 
-  def verifyAccessCode(email: String, accessCode: String): Future[Option[String]] = {
+  def verifyAccessCode(email: String, accessCode: String): Future[Try[String]] = {
```

Comment:
Just because I found it easier to refactor the existing code with a Try[String].
Not to say that it couldn't have been done with Future[String]